pkg/builder/controller.go

```diff
@@ -90,8 +87,9 @@ type ForInput struct {
 
 // For defines the type of Object being *reconciled*, and configures the ControllerManagedBy to respond to create / delete /
 // update events by *reconciling the object*.
+//
 // This is the equivalent of calling
-// Watches(&source.Kind{Type: apiType}, &handler.EnqueueRequestForObject{}).
+// Watches(source.Kind(cache, &Type{}, &handler.EnqueueRequestForObject{}).
```

Comment:
```suggestion
// Watches(source.Kind(cache, &Type{}, &handler.EnqueueRequestForObject{})).
```
if I'm counting correctly :)

Reply:
fixed